src/objects/Task__c.object

```diff
@@ -0,0 +1,322 @@
+        <writeRequiresMasterRead>false</writeRequiresMasterRead>
+    </fields>
+    <fields>
+        <fullName>Description__c</fullName>
+        <externalId>false</externalId>
+        <label>Description</label>
+        <length>32768</length>
+        <trackTrending>false</trackTrending>
+        <type>LongTextArea</type>
+        <visibleLines>3</visibleLines>
+    </fields>
+    <fields>
+        <fullName>Difficulty__c</fullName>
+        <externalId>false</externalId>
+        <label>Difficulty</label>
+        <picklist>
+            <picklistValues>
+                <fullName>Trivia</fullName>
+                <default>false</default>
+            </picklistValues>
+            <picklistValues>
+                <fullName>Easy</fullName>
+                <default>false</default>
+            </picklistValues>
+            <picklistValues>
+                <fullName>Medium</fullName>
+                <default>false</default>
+            </picklistValues>
+            <picklistValues>
+                <fullName>Hard</fullName>
+                <default>false</default>
+            </picklistValues>
+            <restrictedPicklist>true</restrictedPicklist>
+            <sorted>false</sorted>
+        </picklist>
+        <required>true</required>
+        <trackTrending>false</trackTrending>
+        <type>Picklist</type>
+    </fields>
+    <fields>
+        <fullName>Due_Date__c</fullName>
+        <externalId>false</externalId>
+        <label>Due Date</label>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Date</type>
+    </fields>
+    <fields>
+        <fullName>Points_Awareded__c</fullName>
+        <externalId>false</externalId>
+        <formula>IF((ISPICKVAL(Status__c,&quot;Completed&quot;)&amp;&amp; RecordType.Name==&quot;TO-DO&quot; &amp;&amp; ISPICKVAL(Difficulty__c,&quot;Trivia&quot;)), 1, IF((ISPICKVAL(Status__c, &quot;Completed&quot;)&amp;&amp; RecordType.Name==&apos;TO-DO&apos; &amp;&amp; ISPICKVAL(Difficulty__c,&apos;Easy&apos;)), 2, IF((ISPICKVAL(Status__c, &quot;Completed&quot;)&amp;&amp; RecordType.Name == &quot;TO-DO&quot; &amp;&amp; ISPICKVAL(Difficulty__c,&quot;Medium&quot;)), 3, IF((ISPICKVAL(Status__c, &quot;Completed&quot;)&amp;&amp; RecordType.Name==&quot;TO-DO&quot; &amp;&amp; ISPICKVAL(Difficulty__c,&quot;Hard&quot;)), 5, IF((ISPICKVAL(Status__c, &quot;Completed&quot;)&amp;&amp; RecordType.Name==&quot;Daily&quot; &amp;&amp; ISPICKVAL(Difficulty__c,&quot;Trivia&quot;)), 0.5, IF((ISPICKVAL(Status__c, &quot;Completed&quot;)&amp;&amp; RecordType.Name==&quot;Daily&quot; &amp;&amp; ISPICKVAL(Difficulty__c,&quot;Easy&quot;)), 1, IF((ISPICKVAL(Status__c, &quot;Completed&quot;)&amp;&amp; RecordType.Name== &quot;Daily&quot; &amp;&amp; ISPICKVAL(Difficulty__c,&quot;Medium&quot;)), 1.5, IF((ISPICKVAL(Status__c, &quot;Completed&quot;) &amp;&amp; RecordType.Name == &quot;Daily&quot;&amp;&amp; ISPICKVAL(Difficulty__c,&quot;Hard&quot;)), 2.5, null))))))))</formula>
```

Comment:
the amount of points a task is going to give once completed does not depend on the Status to be "Completed" so please remove that as a condition.